playbooks/vmware-workflow/restore_vm.yml

```diff
@@ -14,16 +14,19 @@
         password: "{{ cohesity_password }}"
         validate_certs: "{{ cohesity_validate_certs }}"
         name: "Testing"
-        job_name: "protect_vm"
+        job_name: "null-279"
         vm_names:
-          - chandra-pwsh01
-        network_name: "VLAN21"
-        vm_folder_name: "cohesity_folder"
+          - Ubuntu
+        #network_name: "VLAN21"
+        #vm_folder_name: "cohesity_folder"
         wait_for_job: true
         state: "present"
-        # restore_to_source: "myvmware.host.com"
-        resource_pool_name: "cohesity-resource_pool"
-        datastore_name: "cohesity_datastore"
+        restore_to_source: true
+        prefix: test_
+        resource_pool_name: VCD RPool
+        datastore_name: cohesity_int_21046_228e3
         network_connected: false
-        interface_group_name: "iface_name"
-        endpoint: vcenter.domain.com
+        #interface_group_name: "iface_name"
+        ##datacenter: A
+        cluster_compute_resource: aa
+        endpoint: vc-67.eco.eng.cohesity.com
```

Comment:
Remove infra endpoints! never expose this in github PRs! 
Let this be the last time, we are bringing this up.